scripts/ethereum/init.sh

```diff
@@ -6,7 +6,7 @@ export GANACHE_PORT="8545"
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-docker pull "alpine"
+# Don't pull before run. We can use whatever version of alpine is available locally.
```

Comment:
How about a conditional pull if there's no local version?

Reply:
That happens automatically in `docker run`